src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/customize_panel/customize_panel_action.tsx

```diff
@@ -79,9 +79,15 @@ export class CustomizePanelAction implements Action<CustomizePanelActionContext>
       (embeddable as VisualizeEmbeddable).getOutput().visTypeName === 'markdown';
 
     const isImage = embeddable.type === 'image';
+    const isNavigation = embeddable.type === 'navigation';
 
     return Boolean(
-      embeddable && hasTimeRange(embeddable) && !isInputControl && !isMarkdown && !isImage
+      embeddable &&
+        hasTimeRange(embeddable) &&
+        !isInputControl &&
+        !isMarkdown &&
+        !isImage &&
+        !isNavigation
```

Comment:
This ensures that the custom time range does not show up in the panel settings flyout for the navigation embeddable:

https://github.com/elastic/kibana/assets/8698078/a5a4397f-8591-4a5f-9e7c-69809eecdd65